resources/views/argument-comments.blade.php

```diff
@@ -22,16 +32,26 @@
         />
 
         <div class="max-w-[900px] mt-6 md:mt-9 mx-auto">
-            <x-argument-card.comment-form
-                :user="$user"
-                :argument="$argument"
-            />
+            <h2 class="text-lg lg:text-2xl font-bold text-gray-900 dark:text-white opacity-80 mb-2">
+                Comments ({{ $argument->comments->count() }})
+            </h2>
 
-            <div class="space-y-4 mt-6 md:mt-8">
+            <div class="space-y-4">
                 @foreach($argument->comments as $comment)
                     <x-argument-card.comment :comment="$comment" />
                 @endforeach
             </div>
+
+            <div class="mt-6  md:mt-8">
+                <h2 class="text-lg lg:text-2xl font-bold text-gray-900 dark:text-white opacity-80 mb-2">
+                    Add a new comment
+                </h2>
+
+                <x-argument-card.comment-form
```

Comment:
You want form to be under all of the comments? I've noticed that most websites make the form above comments since why do people need to scroll god knows how many comments just to make a comment. What if an argument get attention and gets like 20 comments? I just think that putting the form on top of comments is better, but it's up to you